### ISSUE TITLE
Feature Request: Adding two buttons for 10 seconds forward and back in the player

### ISSUE THREAD

**Issue by jekthewarrior** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3690]
I have been using the player for a while now and I wanted to know if it would be possible to get two buttons for 10 seconds forward and back in the player? This would make it more convenient for the user to jump around a track, especially if that track is a single music compilation on YouTube Music. It would add another layer of functionality that would be beneficial to everyone using the player.

Preferably, it would be located near the central controls near the track control buttons and would be able to be used with hotkeys as the other controls are.

If the buttons were pressed and held, it would successively skip 10 seconds forward or back. If the 10 seconds are skipped back or forward at the first or last 10 seconds of the current playing track, then it would just skip to the previous or next track. This is how I would envision these buttons should work.

**Comment by IndefiniteBen** [comment]
I would also like a hotkey combination that can skip a fixed number of seconds forward, which I don't see as a big thing. However, pressing and holding the skip track buttons to trigger this? I think that's a more difficult thing, refer to #2364 and #3293 for some of the issues correctly interpreting media keys.

**Comment by jekthewarrior** [comment]
Makes sense to me, but again, it's only a suggestion. As long as a skip function with a hopefully customizable length is created, I will be satisfied.
